unified-codes/apps/web/src/components/containers/error/ErrorPage.tsx

```diff
@@ -1,34 +1,63 @@
+
 import * as React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { makeStyles, createStyles } from '@unified-codes/ui/styles';
+
+import ErrorNotFoundTop from './ErrorNotFoundTop';
+import ErrorNotFoundCenter from './ErrorNotFoundCenter';
+import ErrorNotFoundBottom from './ErrorNotFoundBottom';
+
+import ErrorLayout from '../../layout/ErrorLayout';
+
+import { Errors, IErrorRouteParams } from '../../../types';
+import { ITheme } from '../../../styles';
+
+const ERROR_CODES = {
+  [Errors.PageNotFound]: '404',
+}
 
-import { Grid } from '@unified-codes/ui/components';
+const useStyles = makeStyles((_: ITheme) => createStyles({
+  root: {
+      flexDirection: 'column',
+      alignContent: 'center',
+      padding: '20px 0 20px 0',
+  },
+  centerContainer: {
+      flexDirection: 'row',
+      justifyContent: 'center',
+  },
+  topContainer: {
+      textAlign: 'center'
+  },
+  bottomContainer: {
+    textAligh: 'center'
```

Comment:
it makes me think of a `sigh`.. should really be changed though. The text alignment on the bottom is not centered

Reply:
3b2705216e6e80f418c76bfa4ee81f2d4834e3c7.